### PR TITLE
GO DHCPD

### PR DESCRIPTION
**Pull request by fdurand**
# Description
GO DHCPD server

# Impacts
dhcp flow

# NEW Package(s) required
A lot

# Delete branch after merge
NO

# NEWS file entries
## New Features
* Go dhcpd server
* Ip per role in inline network

### CODE REVIEW

**Comment by extrafu** [comment]
Should we close this one and re-open a new PR with all the services in Golang?

File: conf/iptables.conf.example [hunk 1]
```diff
@@ -87,9 +95,6 @@
 # DNS
 -A input-internal-vlan-if --protocol tcp --match tcp --dport 53  --jump ACCEPT
 -A input-internal-vlan-if --protocol udp --match udp --dport 53  --jump ACCEPT
-# DHCP
--A input-internal-vlan-if --protocol udp --match udp --dport 67  --jump ACCEPT
```
Comment (julsemaan) [inline conf/iptables.conf.example:91]: Why don't we accept those anymore ?

Comment (fdurand) [reply conf/iptables.conf.example:91]: It's managed by iptables.pm

**Comment by extrafu** [comment]
Should we close that PR?

**Pull request closed without merging**